lte/gateway/python/scripts/pipelined_cli.py

```diff
@@ -64,6 +64,130 @@
     ],
 )
 
+UEInfo5G = namedtuple(
+    'UEInfo5G', [
+        'subscriber_id', 'in_teid', 'out_teid', 'ue_ip_addr',
+        'gnb_ip_addr', 'del_rule_id', 'add_rule_id', 'ipv4_dst', 'allow', 'priority',
+    ],
+)
+
+
+def _gen_ue_set5g(number_of_iterations):
```

Comment:
It looks to me like the only difference from the normal get_ue_set is the tunnels? In that case just add in/out teid into the UEInfo

Reply:
@koolzz , Mehul has addressed the comment and fixed it. Please review it.